README.md

```diff
@@ -60,25 +76,41 @@ spec:
     type: OpenShiftSDN
   serviceNetwork: 172.30.0.0/16
 ```
+For OVN it is:
+```yaml
+apiVersion: networkoperator.openshift.io/v1
+kind: NetworkConfig
+metadata:
+  name: default
+spec:
+  additionalNetworks: null
+  clusterNetworks:
+  - cidr: 10.128.0.0/14
+    hostSubnetLength: 9
+  defaultNetwork:
+    type: OVNKubernetes
+  serviceNetwork: 172.30.0.0/16
+```
+The cidr, hostSubnetLength is internally converted into the cidr/hostPrefix etended cidr used by OVN, 10.128.0.0/14/23.
 
 ## Configuring IP address pools
-Users must supply at least two address pools - one for pods, and one for services. These are the ClusterNetworks and ServiceNetwork parameter. Some network plugins, such as OpenShiftSDN, support multiple ClusterNetworks. All address blocks must be non-overlapping. You should select address pools large enough to fit your anticipated workload.
+Users must supply at least two address pools - one for pods, and one for services. These are the ClusterNetworks and ServiceNetwork parameter. Some network plugins, such as OpenShiftSDN, and OVNKubernetes, support multiple ClusterNetworks. All address blocks must be non-overlapping. You should select address pools large enough to fit your anticipated workload. Each pool must be able to hold 1 or more hostPrefix allocations.
 
 For future expansion, multiple `serviceNetwork` entries are allowed by the configuration but not actually supported by any network plugins. Supplying multiple addresses is invalid.
 
-Each `clusterNetwork` entry has an additional required parameter, `hostPrefix`, that specifies the address size to assign to assign to each individual node.  For example,
+Each `clusterNetwork` entry has an additional required parameter, `hostPrefix`, that specifies the address size to assign to each individual node.  For example,
```

Comment:
(This PR is not going to merge before 4.0 branches, so "bugfixes" like this and non-OVN-specific clarifications like the "hostPrefix allocations" comment above should be split off into a separate PR.)